doc/howto/import_machines_to_instances.md

```diff
@@ -161,6 +161,115 @@ Complete the following steps to migrate an existing machine to a LXD instance:
    Instance foo successfully created
    ```
 
+   </details>
+   <details>
+   <summary>Expand to see an example output for importing to a VM</summary>
+
+   ```{terminal}
+   :input: sudo ./bin.linux.lxd-migrate
+
+   Please provide LXD server URL: https://192.0.2.7:8443
```

Comment:
Could we use 127.0.0.1 here to show how to import a VM into the local LXD server.

Also does this guide show how to configure LXD to listen on the network or 127.0.0.1?

Reply:
I added a note and a link to the instructions for exposing the server - that's not part of this how-to.

Also, importing to the local machine isn't really the main purpose of the tool, so I'd rather stick to a random IP for the example.

Reply:
I've asked @grebennikov to take a look at this to see if this helps in his use-case. Thanks

Reply:
@grebennikov has asked that we:

1. Explain what image format that needs to be provided (a raw disk image file).
2. Expand on options to configure networking for the imported VM.

Reply:
I added a few links and a note that clarifies that you need to either set up your profile(s)/network/storage before you start the migration process or configure your new instance after migration.
Maybe that makes things clearer.

Reply:
Thanks @ru-fu !